packages/suite/src/hooks/wallet/useSendFormCompose.ts

```diff
@@ -159,13 +190,41 @@ export const useSendFormCompose = ({
             const { setMaxOutputId } = values;
             // set calculated and formatted "max" value to `Amount` input
             if (typeof setMaxOutputId === 'number' && composed.max) {
-                setAmount(setMaxOutputId, composed.max);
+                const networkDecimals = NETWORKS.find(network => network.symbol === account.symbol)
+                    ?.decimals as number;
+
+                setAmount(
```

Comment:
instead of making changes in this hook you can format the amount in sendFormBitcoinActions.composeTransaction:

- `getBitcoinComposeOutputs` function is converting BTC > SAT anyway (beause utxo-lib is making calculation in satoshi) so it could be done conditionally there
- `formatNetworkAmount` in processing response is converting back SAT > BTC, so again it could be done conditionally


if you do this like that then the all changes done in this hook (like adding new `useDidUpdate` sub hook, conditions, formatting, etc) will not be needed at all

Reply:
b1097cad2be09fc04569fd0ecbda3f1632d2d09a